application/cms/models.py

```diff
@@ -1019,6 +1020,13 @@ class Topic(db.Model):
     def has_published_measures(self):
         return any(subtopic.has_published_measures for subtopic in self.subtopics)
 
+    @property
+    def short_title_or_title(self):
```

Comment:
I'm not sure this is a thing that actually provides much value? You can just write `self.short_title or self.title` inline wherever you want this.

Reply:
Unless Jinja doesn't allow that, in which case, fine. But maybe a comment saying that's what it's for, if true, would be useful.

Reply:
@samuelhwilliams it could quite easily have been in the template, but I thought perhaps it’d get re-used elsewhere (eg in the dashboard views), and maybe we should be aiming to avoid having much logic in the views anyway? But 🤷‍♂️

Reply:
Maybe you're right. I wouldn't mind sitting down with you and getting a lesson in this paradigm you're promoting. I've heard of it plenty but not used to implementing it.

Reply:
@samuelhwilliams "paradigm" is probably overstating it... I think it's generally worth trying to start moving logic to object methods though. Easier to understand and test.

Reply:
Maybe? Or do you end up with duplicated tests - tests for the property on the model, and then tests rendering the view to ensure you've *used* the property and it can display both options? Or do you not test the view in that case?

Reply:
Depends how comprehensive you want to be, I guess. You might have more comprehensive tests on the model (all possible options) and a single test for the integration test?